groups/sig-architecture/groups.yaml

```diff
@@ -0,0 +1,18 @@
+groups:
+
+  #
+  # sig-architecture infra acces
+  #
+  # Each group here represents privileged access to some set of infrastructure
+  # related to subprojects owned by sig-architecture
+  #
+
+  - email-id: k8s-infra-keps@kubernetes.io
+    name: k8s-infra-keps
+    description: |-
+      ACL for access to KEP related infrastructure
+    settings:
+      ReconcileMembers: "true"
+    members:
```

Comment:
SIG leads + KEP tool reviewers/approvers?

Reply:
I want to know who specifically is signing up to own and maintain this tooling and infrastructure. Not just a blanket list of people

Reply:
@sftim @shekhar-rajak
Do you have suggestions for this list as originators of the request?

@justaugustus
Are you suggesting https://github.com/kubernetes/enhancements/blob/561c1d969a7dc02d1f17b352d0e2b31953a31c53/OWNERS_ALIASES#L164-L170 ? Not sure if I've got e-mails for everyone there